### PR TITLE
Adds support for bitwise `AND` / `OR` on numbers 

### LINKED ISSUE #16: [v2] add bitwise operators
much like v1, v2 should support bitwise operators such as 

`|` `&`

### PR DESCRIPTION
**Pull request by scrofungulus**
Closes #16

### CODE REVIEW

File: samples/bitwise.br [hunk 1]
```diff
@@ -0,0 +1,5 @@
+let x = 5 & 1;
```
Comment (TheDanBlanco) [inline samples/bitwise.br:1]: instead of adding a sample, add a test or two in the tests package

**Comment by TheDanBlanco** [comment]
i don't have coverage wired up in github yet. what's the coverage % on these features? you can check by running `make coverage`

**Review by TheDanBlanco — CHANGES_REQUESTED** [review]

File: samples/bitwise.br (hunk 1 above)
Comment (scrofungulus) [reply samples/bitwise.br]: done

**Comment by scrofungulus** [comment]
image
@TheDanBlanco lmk if this is better or worse i don't have a reference point

**Review by TheDanBlanco — APPROVED** [review]
lgtm, thanks for the pr

**Pull request merged**